quickwit/quickwit-indexing/src/actors/merge_planner.rs

```diff
@@ -141,8 +157,18 @@ impl MergePlanner {
 
                 for merge_operation in merge_operations {
                     info!(merge_operation=?merge_operation, "Planned merge operation.");
-                    ctx.send_message(&self.merge_split_downloader_mailbox, merge_operation)
-                        .await?;
+                    let tracked_merge_operations = self
+                        .ongoing_merge_operations_inventory
+                        .track(merge_operation);
+                    ctx.send_message(
+                        &self.merge_split_downloader_mailbox,
+                        tracked_merge_operations,
+                    )
+                    .await?;
+                    INDEXER_METRICS
+                        .ongoing_num_merge_operations_total
+                        .with_label_values(&[&self.pipeline_id.index_id])
+                        .set(self.ongoing_merge_operations_inventory.list().len() as i64);
```

Comment:
We only set it here... Which means that counter can stay up for a no reason at all for a long time.

Reply:
Maybe we should have another loop to refresh the counter ?

Reply:
Indeed :/.